dist/api_data.py

```diff
@@ -120,6 +120,14 @@ def __cmp__(self, other):
 
 # Per-file configuration
 file_config = format_meta + [
+    Config('access_pattern_hint', 'none', r'''
+        It is recommended that workloads that consist primarily of
+        updates and/or point queries specify \c random.  Workloads that
+        do many cursor scans through large ranges of data specify
+        \c sequential and other workloads specify \c none.  The
+        option leads to an advisory call to an appropriate operating
+        system API where available''',
+        choices=['none', 'random', 'sequential']),
```

Comment:
We're making a change to the default behavior here (from "random" to "none").  I suspect that's right but would like some data to back it up.  I'll kick off MongoDB perf tests against this branch to see if there are any regressions there...